onnx/defs/traditionalml/defs.cc

```diff
@@ -814,108 +854,106 @@ ONNX_ML_OPERATOR_SET_SCHEMA(
     1,
     OpSchema()
         .SetDoc(TreeEnsembleRegressor_ver1_doc)
-        .Input(0, "X", "Input N,F", "T")
+        .Input(0, "X", "Input of shape [N,F]", "T")
         .Output(0, "Y", "N classes", "tensor(float)")
         .TypeConstraint(
             "T",
             {"tensor(float)",
              "tensor(double)",
              "tensor(int64)",
              "tensor(int32)"},
-            " allowed types.")
+            "")
         .Attr(
             "nodes_treeids",
-            "tree id for this node",
+            "Tree id for each node.",
             AttributeProto::INTS,
             OPTIONAL)
         .Attr(
             "nodes_nodeids",
-            "node id for this node, node ids must restart at zero for each tree and increase sequentially.",
+            "Node id for each node. Ids ids must restart at zero for each tree and increase sequentially.",
             AttributeProto::INTS,
             OPTIONAL)
         .Attr(
             "nodes_featureids",
-            "feature id for this node",
+            "Feature id for each node.",
             AttributeProto::INTS,
             OPTIONAL)
         .Attr(
             "nodes_values",
-            "thresholds to do the splitting on for this node.",
+            "Thresholds to do the splitting on for each node.",
             AttributeProto::FLOATS,
             OPTIONAL)
         .Attr(
             "nodes_hitrates",
-            "popularity of the node, used for performance and may be omitted",
+            "Popularity of each node, used for performance and may be omitted.", 
             AttributeProto::FLOATS,
             OPTIONAL)
         .Attr(
             "nodes_modes",
-            "enum of behavior for this node as enum of BRANCH_LEQ, BRANCH_LT, BRANCH_GTE, BRANCH_GT, BRANCH_EQ, BRANCH_NEQ, LEAF",
+            "The node kind, that is, the comparison to make at the node. There is no comparison to make at a leaf node.<br>One of 'BRANCH_LEQ', 'BRANCH_LT', 'BRANCH_GTE', 'BRANCH_GT', 'BRANCH_EQ', 'BRANCH_NEQ', 'LEAF'",
```

Comment:
original description contains the enum strings. Keep them please so that readers will know what are the potential string values for the attribute.

Reply:
They are still there. The line got long, but the values are there.